tools/urls-check

```diff
@@ -32,7 +32,7 @@ import task
 
 sys.dont_write_bytecode = True
 
-DAYS = 3
+DAYS = 7
```

Comment:
This change should  go into the previous commit. It's also fine to add it to .tasks in the same commit (but I have no strong opinion either way).